opendatahub-io/odh-dashboard
# Add group check for cluster-admin

## Description

Fixes #464

## How Has This Been Tested?



1. Create a new group
2. Add a cluster-admin rolebinding
3. Add a user to that group
4. The user should have cluster-admin roles
5. Log in to the dashboard
6. The user **should have** cluster-admin permission in the dashboard

## Merge criteria:
